cruise-control/src/test/resources/basic-auth.credentials

```diff
@@ -0,0 +1,2 @@
+ccadmin: pass123,ADMIN
+ccuser: pwd123,USER
```

Comment:
Nit: To make it more obvious to the neglectful user that these are used only for unit tests, should we add a `Test` prefix to username and password?

`ccadmin` -> `ccTestAdmin`
`ccuser ` -> `ccTestUser`
`pass123` -> `TestPass123`
`pwd123` -> `TestPwd123`